dist/templates/k8s.ovn.org_egressfirewalls.yaml.j2

```diff
@@ -23,23 +23,15 @@ spec:
     name: v1
     schema:
       openAPIV3Schema:
-        description: EgressFirewall describes the current egress firewall for a Namespace.
-          Traffic from a pod to an IP address outside the cluster will be checked
-          against each EgressFirewallRule in the pod's namespace's EgressFirewall,
-          in order. If no rule matches (or no EgressFirewall is present) then the
-          traffic will be allowed by default.
+        description: EgressFirewall describes the current egress firewall for a Namespace. Traffic from a pod to an IP address outside the cluster will be checked against each EgressFirewallRule in the pod's namespace's EgressFirewall, in order. If no rule matches (or no EgressFirewall is present) then the traffic will be allowed by default.
```

Comment:
Is there any issue on the doc side if you keep the space on the description?

Reply:
There is no issues on the doc side that I can see from `kubectl explain egressfirewalls` or `kubectl describe crd egressfirewall`